pkg/features/list.go

```diff
@@ -133,4 +133,11 @@ var (
 
 	// Flattens CVE Data Model for improved accuracy and performance
 	FlattenCVEData = registerFeature("Uses a flattened CVE Data Model improved accuracy and performance", "ROX_FLATTEN_CVE_DATA")
+
+	// ScannerV4SkipLanguage disables any language-level packages support.
```

Comment:
Feature flags enable features, not disable.
```suggestion
	// ScannerV4SkipLanguage enables any language-level packages support.
```